pt_miniscreen/pages/root/projects.py

```diff
@@ -51,52 +66,87 @@ def from_file(cls, file):
             raise InvalidConfigFile(e)
 
 
+def get_project_environment():
+    env = os.environ.copy()
+
+    if "PT_MINISCREEN_SYSTEM" in env:
+        # Allow to take over miniscreen
+        env.pop("PT_MINISCREEN_SYSTEM")
+
+    first_display = get_first_display()
+    if first_display is not None:
+        env["DISPLAY"] = first_display
+
+    # Print output of commands in english
+    env["LANG"] = "en_US.UTF-8"
```

Comment:
is this necessary in this context? we're not parsing the output of people's projects, I think best to leave it in the language they have chosen